file/1.5/modules/ROOT/pages/file-copy-move.adoc

```diff
@@ -1,31 +1,64 @@
 = Copy and Move Files - Mule 4
 :page-aliases: connectors::file/file-copy-move.adoc
 
-The File connector can copy or move files or directories on demand. If you specify the `sourcePath` as a directory name,
-a directory is copied or moved. Otherwise if `sourcePath` is a file, only the file is copied or moved.
-If a file or directory already exists and `overwrite` is `false`, the `FILE:FILE_ALREADY_EXISTS` error occurs.
+Anypoint Connector for File (File Connector) provides the *Copy* and *Move* operations that enable you to copy or move files or directories on demand. 
+
+* *Source path* +
+The directory path name or file name to copy or move. If a file or directory already exists and the *Overwrite* field is set to `false`, the connector throws the message `FILE:FILE_ALREADY_EXISTS`.
+
+* *Target path* +
+ The directory path name into which to copy or move the file. If a target path doesn't exist, and neither does its parent, the connector makes an attempt to create the path depending on the value of the *Create parent directories* argument. If *Create parent directories* is `false`, then `FILE:ILLEGAL_PATH` occurs.
+
+* *Rename to* +
+Renames the file as part of the operation. Set this field with file name, not a path. The connector retains the original file name, if you do not configure this field.
```

Comment:
Suggestion here and wherever else applicable:
Set this field with a file name instead of a path.